File: 백준/Bronze/30794. 가희와 클럽 오디션 1/가희와 클럽 오디션 1.cs
string[] input = Console.ReadLine().Split();
int lv = int.Parse(input[0]);
string sign = input[1];

if (sign == "miss")
    Console.WriteLine(0);
else if (sign == "bad")
    Console.WriteLine(200 * lv);
else if (sign == "cool")
    Console.WriteLine(400 * lv);
else if (sign == "great")
    Console.WriteLine(600 * lv);
else if (sign == "perfect")
    Console.WriteLine(1000 * lv);

File: 백준/Bronze/30794. 가희와 클럽 오디션 1/가희와 클럽 오디션 1.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int lv = sc.nextInt();
        String sign = sc.next();

        if (sign.equals("miss"))
            System.out.println(0);
        else if (sign.equals("bad"))
            System.out.println(200 * lv);
        else if (sign.equals("cool"))
            System.out.println(400 * lv);
        else if (sign.equals("great"))
            System.out.println(600 * lv);
        else if (sign.equals("perfect"))
            System.out.println(1000 * lv);

        sc.close();
    }
}
